bazarr/main.py

```diff
@@ -2215,4 +2218,4 @@ def api_help():
         args.port) if args.port else str(settings.general.port)) + str(base_url))
     server.start()
 except KeyboardInterrupt:
-    shutdown()
+    doShutdown()
```

Comment:
Basically, if authorizations were enabled the shutdown would not work and leave the server in an awkward state: everything still running except the web part